inspire_json_merger/pre_filters.py

```diff
@@ -216,3 +217,11 @@ def remove_duplicated_titles(root, head, update):
 
 filter_documents_same_source = partial(keep_only_update_source_in_field, 'documents')
 filter_figures_same_source = partial(keep_only_update_source_in_field, 'figures')
+
+
+def normalize_title(data):
```

Comment:
I don't think this is the right place for it, `pre_filters` are functions that are applied to the record before passing it on to the merger. Here it's used in a comparator configuration, which is something completely different. I would just leave it in `comparators.py`.